dali/pipeline/operators/decoder/nvjpeg_decoder_test.cc

```diff
@@ -111,5 +117,37 @@ TYPED_TEST(nvjpegDecodeTest, TestBatchedPNGDecode4T) {
   this->PngTestDecode(true, 4);
 }
 
+TYPED_TEST(nvjpegDecodeTest, TestSingleTiffDecode) {
```

Comment:
How about testing all files mixed together?

Reply:
I'll do it

Reply:
OK, I see that is not that easy now. Let postpone it now.

Reply:
I think that the easiest way to create the test with the mixed set of images would be the following one:
1. Create new class
```
template <typename ImgType>
class nvjpegDecodeTestMixed : public nvjpegDecodeTest
public:
   void SetUp() override {
    DALISingleOpTest::SetUp();
    jpegs_.LoadImages(image_folderJPEG);
    jpegs_.LoadImages(image_folderPNG);
    jpegs_.LoadImages(image_folderTIFF);
    }
 uint32_t GetImageLoadingFlags() const  override {
    return t_undefinedImgType; 
  }
```

2. Remaining part is
```
TYPED_TEST_CASE(nvjpegDecodeTestMixed, Types);

TYPED_TEST(nvjpegDecodeTestMixed, TestSingleMixedDecode) {
  this->TestDecode(false, 1);
}
. . .
```